src/components/PageTitle.tsx

```diff
@@ -49,3 +51,7 @@ const PageHeader = styled.div`
   display: flex;
   justify-content: space-between;
 `;
+
+const TitleWrapper = styled.div`
+  width: 60%;
```

Comment:
I think this may be over restricting the title width: 
example from code change:
<img width="1664" alt="Screen Shot 2020-05-12 at 1 19 27 PM" src="https://user-images.githubusercontent.com/10734386/81725115-5eafd700-9453-11ea-8c54-14988f4b29ab.png">

design:
<img width="1440" alt="Screen Shot 2020-05-12 at 1 19 06 PM" src="https://user-images.githubusercontent.com/10734386/81725111-5ce61380-9453-11ea-98c3-b1a6db54c601.png">